package.json

```
@@ -6,6 +6,8 @@
     "build": "prisma generate && babel src -d dist --copy-files",
     "start": "prisma generate && yarn migrate && nodemon --exec babel-node -e js,graphql src/index.js",
     "serve": "yarn migrate && node dist/index.js",
+    "workers": "node dist/workers/index.js",
```

```suggestion
    "worker": "node dist/workers/index.js",
```
`yarn start worker`?

---

Right now we only have one worker, but after this PR we're going to be adding others there, so I thought it made sense to leave as `workers`.

---

JFYI typically when we say worker it means 1 process, so you run 1 worker (1 process) with few subscriptions o different subjects. So IMO it should be `worker`.
wdyt @schnie ?

---

Ok, that makes sense because we're not running each PubSub as a separate worker.  I can switch this to `worker`.